data_diff/dbt.py

```diff
@@ -202,18 +205,25 @@ def _local_diff(diff_vars: DiffVars) -> None:
         rich.print(diff_output_str)
         return
 
-    mutual_set = set(table1_columns) & set(table2_columns)
-    table1_set_diff = list(set(table1_columns) - set(table2_columns))
-    table2_set_diff = list(set(table2_columns) - set(table1_columns))
+    column_set = set(table1_columns).intersection(table2_columns)
+    table1_diff = set(table1_columns).difference(table2_columns)
+    table2_diff = set(table2_columns).difference(table1_columns)
```

Comment:
Out of curiosity, why? The original version with `&` & `-` seemed more readable.